src/helpers/citySelectionHelper.ts

```diff
@@ -0,0 +1,21 @@
+import { addToStore, readFromStore, removeFromStore } from '../helpers';
+
+export const SELECTED_CITY = 'selectedCity';
+
+export const loadStoredCity = async () => {
+  const city = await readFromStore(SELECTED_CITY);
+  return city;
+};
+
+export const storeCity = async (city: string | null) => {
+  if (city) {
+    await addToStore(SELECTED_CITY, city);
+    return city;
+  }
+  return null;
+};
+
+export const resetCity = async () => {
+  await removeFromStore(SELECTED_CITY);
+  return null;
+};
```

Comment:
did you check our existing helpers?
have am look at `voucherHelper` for example:

```
export const storeVoucherMemberId = (memberId?: string) => {
  if (memberId) {
    addToStore(VOUCHER_MEMBER_ID, memberId);
  } else {
    removeFromStore(VOUCHER_MEMBER_ID);
  }
};
```

this is the way we used to implement adding and removing